code/datums/supplypacks.dm

```diff
@@ -1382,27 +1382,37 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = list(access_kitchen)
 	group = "Hospitality"
 
-/datum/supply_packs/randomised/assorted_meats
-	name = "Assorted meats"
-	num_contained = 10
+/datum/supply_packs/randomised/premium_meats
+	name = "Premium meats"
+	num_contained = 8
 	contains = list(/obj/item/weapon/reagent_containers/food/snacks/meat/mimic,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/bearmeat,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/spidermeat,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/xenomeat,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/carpmeat/imitation,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/syntiflesh,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/spiderleg,
-					/obj/item/weapon/reagent_containers/food/snacks/spidereggs,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/diona,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat,
-					/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken/raw_vox_chicken,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/nymphmeat,
+					/obj/item/weapon/reagent_containers/food/snacks/meat/crabmeat,
 					/obj/item/weapon/reagent_containers/food/snacks/meat/polyp,
+					/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken/raw_vox_chicken)
+	cost = 30
+	containertype = /obj/structure/closet/crate/secure/basic
+	containername = "Premium meats crate"
+	access = list(access_kitchen)
+	group = "Hospitality"
+
+/datum/supply_packs/randomised/assorted_meats
+	name = "Assorted meats"
+	num_contained = 8
+	contains = list(/obj/item/weapon/reagent_containers/food/snacks/meat/rawchicken,
```

Comment:
Not sure normal ass chicken deserves to be in with the gunkmeats

Reply:
Synth meat is there too, it's supposed to be non-premium meat, stuff you get for cheap. Regular chicken and synth meat fit the bill.